views/group/index.pug

```diff
@@ -18,10 +18,11 @@ block content
                 else
                   i.fas.fa-lock-open
             p Szint: #{group.room}
-            p= group.description
+            //TODO: description must be sliced if too long
```

Comment:
A better thing to do is to check on the backend for length. I'm currently working on it so you shouldn't bother.